dev-docs/bidders/dichange.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: bidder
+title: dichange
+description: dichange Bidder Adapter
+biddercode: dichange
+media_types: banner, video
```

Comment:
Could you add

```
safeframes_ok: true/false
deals_supported: true/false
fpd_supported: true/false
```

meta data as well, please :blush:

Reply:
Thanks for the comment. Please let me know if everything is correct now